src/twisted/python/failure.py

```diff
@@ -417,11 +417,18 @@ def __init__(self, exc_value=None, exc_type=None, exc_tb=None, captureVars=False
                 )
             )
             tb = tb.tb_next
+
+    @property
+    def parents(self):
```

Comment:
I know that in `trunk` we alrady have `Failure.parents` as public API and without documentation.

Can we add a few words about `Failure.parents` in this PR?

The documentation should go in `Failure.__doc__` and not here... but I can't leave a comment there.

Reply:
It's an implementation detail, it's better if people _don't_ use it. The fact it's public is a historic quirk...

Reply:
Thanks for the info.

If this should be private, then maybe we need a followup issue to mark it private and raise a deprecation warning for any public usage of the API ?